mobile/src/main/java/rs/readahead/washington/mobile/data/database/DataSource.java

```diff
@@ -645,6 +652,68 @@ private CollectForm getBlankCollectForm(String formID) {
         return forms;
     }
 
+    private List<CollectForm> getFavoriteCollectForms(){
+        Cursor cursor = null;
+        List<CollectForm> forms = new ArrayList<>();
+
+        try {
+            final String query = SQLiteQueryBuilder.buildQueryString(
+                    false,
+                    D.T_COLLECT_BLANK_FORM + " JOIN " + D.T_COLLECT_SERVER + " ON " +
+                            D.T_COLLECT_BLANK_FORM + "." + D.C_COLLECT_SERVER_ID + " = " + D.T_COLLECT_SERVER + "." + D.C_ID,
+                    new String[]{
+                            cn(D.T_COLLECT_BLANK_FORM, D.C_ID, D.A_COLLECT_BLANK_FORM_ID),
+                            D.C_COLLECT_SERVER_ID,
+                            D.C_FORM_ID,
+                            D.T_COLLECT_BLANK_FORM + "." + D.C_NAME,
+                            D.C_VERSION,
+                            D.C_HASH,
+                            D.C_DOWNLOADED,
+                            D.C_UPDATED,
+                            D.C_FAVORITE,
+                            D.C_DOWNLOAD_URL,
+                            cn(D.T_COLLECT_SERVER, D.C_NAME, D.A_SERVER_NAME),
+                            cn(D.T_COLLECT_SERVER, D.C_USERNAME, D.A_SERVER_USERNAME)},
+                    D.C_FAVORITE +" = true " , null, null,
```

Comment:
Actually favorite is stored as int, so this line should be : 
cn(D.T_COLLECT_BLANK_FORM,D.C_FAVORITE) + " = 1" , null, null,